packages/mux-uploader/src/progress.ts

```diff
@@ -0,0 +1,167 @@
+import { globalThis, document } from 'shared-polyfills';
```

Comment:
nitpick(blocking): We should probably keep the module names consistent (aka name it `mux-uploader-progress`.